src/Concordium/Client/LegacyCommands.hs

```diff
@@ -277,7 +277,7 @@ getAccountInfoCommand =
     "GetAccountInfo"
     (info
        (GetAccountInfo <$>
-        strArgument (metavar "IDENTIFIER" <> help "Account address or credential id to be queried about.") <*>
+        strArgument (metavar "IDENTIFIER" <> help "Account address, account index or credential id to be queried about.") <*>
```

Comment:
The `account show` command should also have its documentation (and possibly implementation) updated.